platform/ios/app/MBXOrnamentsViewController.m

```diff
@@ -0,0 +1,92 @@
+#import "MBXOrnamentsViewController.h"
```

Comment:
This example should be moved once this PR lands to https://github.com/mapbox/ios-sdk-examples repo as we use use the latter for manual testing (with Objc / Swift versions).